docs/development/maintainers/releasing.rst

```diff
@@ -84,7 +84,7 @@ The procedure for the feature freeze is as follows:
    the content but leaving the basic structure.  You may also need to
    replace the "by the numbers" numbers with "xxx" as a reminder to update them
    before the next release. Then add the new version to the top of
-   ``docs/whatsnew/index.rst``, update the reference in ``docs/index.rst`` to
+   ``docs/whatsnew/index.rst``, update the reference in ``docs/index.rst`` and ``docs/index_getting_started.rst`` to
```

Comment:
Can you add it to the git add commands two bullets down?

Reply:
Done!